source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -493,6 +493,12 @@ def __init__(self, output_type_serializer):
             type=int,
             help="the additional time added to the taxi section, right before riding the taxi but after hopping off the public transit",
         )
+        parser_get.add_argument(
+            "_pt_planner",
+            type=OptionValue(['kraken', 'loki']),
```

Comment:
Can't use enums? like [here](https://github.com/CanalTP/navitia/blob/dev/source/navitiacommon/navitiacommon/constants.py#L34)

Reply:
I don't really known how to use that !